llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -70,6 +86,50 @@
 log = logging.getLogger(__name__)
 
 
+def _rotary_embedding(config: MPTConfig):
```

Comment:
please don't pass the full config in. just explicitly pass in the pieces you need

Reply:
Done, thanks for the suggestion!